src/systems/actions.cairo

```diff
@@ -17,6 +17,14 @@ mod actions {
         position::{Position, Vec2}, moves::{Moves, Direction, DirectionsAvailable}
     };
 
+    #[derive(Copy, Drop, Serde)]
+    #[dojo::event]
```

Comment:
We also need `#[dojo::model]` to ensure Torii indexes it as an off-chain model. :)